core/rint/src/TRint.cxx

```diff
@@ -757,6 +760,10 @@ Longptr_t  TRint::ProcessLineNr(const char* filestem, const char *line, Int_t *e
          SetPrompt(fNonContinuePrompt);
          fNonContinuePrompt.Clear();
       }
+      std::string_view sv(line);
+      auto lastNonSpace = sv.find_last_not_of(" \t");
+      fBackslashContinue = (lastNonSpace != std::string_view::npos
+                            && sv[lastNonSpace] == '\\');
```

Comment:
Isn't this mis-identifying escaped spaces as newlines?
```c++
$ cat t.cxx
const char c[] = "abc\
def";
```
(which has a space behind `\`) gives
```
$ clang++ -fsyntax-only t.cxx
t.cxx:1:23: warning: backslash and newline separated by space [-Wbackslash-newline-escape]
const char c[] = "abc\
                      ^
1 warning generated.
```
I suppose space-escaping doesn't exist in C++, but `\\ ` does. What's your opinion / recommendation / verdict? :-)

Reply:
> ```
> $ clang++ -fsyntax-only t.cxx
> t.cxx:1:23: warning: backslash and newline separated by space [-Wbackslash-newline-escape]
> const char c[] = "abc\
>                      ^
> ```

According, e.g., to https://gcc.gnu.org/onlinedocs/gcc-3.2.3/cpp/Initial-processing.html, the PP will still get this as a continued line, but usually warn about it, as it might be an editing mistake.  In general, any line ending in `\\` (+ maybe whitespace after it), is a continued line.

> I suppose space-escaping doesn't exist in C++, but \\ does. What's your opinion / recommendation / verdict? :-)

If `\\\\` appears at the end of a line, the last `\\` still means "continued line".  The remaining `\\` is joined to whatever is on the next line, e.g.
```
const char c[] = "abc\\
ndef";
```
is the same as
```
const char c[] = "abc\ndef";
```

And hence, the implementation in this PR should be correct.  :-)